content/en/logs/log_collection/javascript.md

```diff
@@ -250,6 +250,10 @@ signupLogger.setContext({
 })
 ```
 
+## Supported Browsers
+
+The `datadog-logs` library supports all modern desktop and mobile browsers, and legacy browsers IE10 and IE11.
```

Comment:
```suggestion
The `datadog-logs` library supports all modern desktop, mobile, and legacy (IE10 and IE11) browsers.
```

Reply:
About `and legacy (IE10 and IE11)`, IMO it makes it a bit less clear that among legacy browsers, we only support IE10 and IE11.
Maybe we could drop the "legacy browsers" and go with something like:
```
The 'datadog-logs` library supports all modern desktop and mobile browsers. IE10 and IE11 are also supported
```
What do you think?

Reply:
Yeah but in this case, what are modern and legacy browsers? 
Is there an official definition we could refer too?

Reply:
I'm ok with @bcaudan's suggestion.

Reply:
So let's go for it :)